draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -916,49 +916,84 @@ The Holder SHOULD verify the binding between SD-JWT and II-Disclosures Object by
 
 ## Verification by the Verifier when Receiving SD-JWT and Holder-Selected Disclosures JWT {#verifier-verification}
 
-Verifiers MUST follow [@RFC8725] for checking the SD-JWT and, if signed, the
+follow [@RFC8725] for checking the SD-JWT and, if signed, the
 HS-Disclosures JWT.
 
 Verifiers MUST go through (at least) the following steps before
-trusting/using any of the contents of an SD-JWT:
+trusting/using any of the contents of an SD-JWT.
 
- 1. Determine if Holder Binding is to be checked according to the Verifier's policy
+If any of the following steps fail, the input is not valid and
+processing MUST be aborted.
+
+### Step 1: Validate Combined Format for Presentation
```

Comment:
Stepping back a bit, I don't think that mandating that the disclosures go in a JWT is necessary. And maybe not even really appropriate. I was hoping to discuss the general issue/idea during the meeting(s) this week. But it's leading to a lot of unnecessary confusion/complexity in these steps. So it's hard to review or comment without raising it.